src/types.ts

```diff
@@ -105,14 +105,16 @@ export interface Row {
   [key: string]: Cell | string;
 }
 
+type ColAlign = 'inherit' | 'left' | 'center' | 'right' | 'justify';
```

Comment:
We never set this to inherit or justify. only `'left' | 'center' | 'right'` works. Then use it in `TextAlign` as well

Reply:
I'll fix it with the next commit